src/web/app/pages-help/instructor-help-page/instructor-help-panel/instructor-help-panel.component.ts

```diff
@@ -27,6 +31,26 @@ export class InstructorHelpPanelComponent {
     this.isPanelExpandedChange.emit(value);
   }
 
-  constructor(public elementRef: ElementRef) { }
+  constructor(public elementRef: ElementRef,
+              private router: Router,
+              private activatedRoute: ActivatedRoute,
+              private location: Location,
+              private clipboard: Clipboard,
+  ) { }
 
+  copyUrlToClipboard(event: Event): void {
+    // Prevent panel from changing state
+    event.stopPropagation();
+
+    const frontendUrl = window.location.origin;
+    const queryParams = { section: this.section, questionId: this.id };
+    const path = this.router.createUrlTree(
+        [],
+        { relativeTo: this.activatedRoute, queryParams },
+    ).toString();
+    const urlToCopy = frontendUrl + path;
```

Comment:
Look into `navigation.service.ts`, I believe we might be able to reuse one of the functions there.